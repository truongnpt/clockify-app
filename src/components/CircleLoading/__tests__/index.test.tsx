import React from 'react';
import { render } from '@testing-library/react';

import { CircleLoading } from '..';

describe('<CircleLoading  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<CircleLoading />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
