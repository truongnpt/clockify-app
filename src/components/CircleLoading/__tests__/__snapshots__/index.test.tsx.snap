// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CircleLoading  /> should match snapshot 1`] = `
<div
  class="css-1a6hei5"
>
  <span
    class="loading css-67azjw"
  >
    <svg>
      loading.svg
    </svg>
  </span>
  
</div>
`;
